CMakeLists.txt

```
@@ -85,8 +85,8 @@ include(GNUInstallDirs)
 install(TARGETS ${PROJECT_NAME} ${XTRCTL_TARGET}
         RUNTIME DESTINATION bin
         LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib
-        PUBLIC_HEADER DESTINATION include)
+        ARCHIVE DESTINATION lib)
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/include/ DESTINATION include)
```

```suggestion
        ARCHIVE DESTINATION lib
        PUBLIC_HEADER DESTINATION include/xtr)
install(DIRECTORY ${CMAKE_SOURCE_DIR}/include/ DESTINATION include)
```

Suggestion to fix the "Target xtr has PUBLIC_HEADER files but no PUBLIC_HEADER DESTINATION" warning message from CMake.